src/components/molecules/Visualizer/Plugin/types.ts

```diff
@@ -32,12 +32,39 @@ export type Reearth = {
   ) => void;
 };
 
+export type MouseEventProps = {
```

Comment:
```suggestion
export type MouseEvent = {
```

Props is a term of React. But here is not React.